manifests/4.5/logging.openshift.io_clusterlogforwarders_crd.yaml

```diff
@@ -112,18 +116,35 @@ spec:
                           cron authpriv ftp ntp security console solaris-cron     local0
                           local1 local2 local3 local4 local5 local6 local7"
                         type: string
+                      msgID:
+                        description: "MsgID is MSGID part of the syslog-msg header
+                          \n MsgID needs to be specified if using rfc5424"
+                        type: string
                       payloadKey:
                         description: PayloadKey specifies record field to use as payload.
                         type: string
+                      procID:
+                        description: "ProcID is PROCID part of the syslog-msg header
+                          \n ProcID needs to be specified if using rfc5424"
+                        type: string
+                      rfc:
+                        description: "Rfc specifies the rfc to be used for sending
+                          syslog \n Rfc values can be one of:  - RFC3164 (https://tools.ietf.org/html/rfc3164)
+                          \ - RFC5424 (https://tools.ietf.org/html/rfc5424) \n If
```

Comment:
Put an explicit default with `// +kubebuilder:validation:Default` - check the exact syntax, I may not have it right.

Reply:
added